### ISSUE TITLE
Modifier not set when current version is provided

### ISSUE THREAD

**Issue by frzb** [relizaio/versioning#2]
# Example 

Using image `2b01ee20fb02` the `Modifier` is not replaced when providing a current version

```
$ docker run --rm relizaio/versioning -s "0Y.0M.0D.Micro-Modifier" -i dev -a bumpPatch -v "22.03.28.2-dev"
22.03.28.3-Modifier
```
In contrast replacement works **without** providing a **current version**

```
$ docker run --rm relizaio/versioning -s "0Y.0M.0D.Micro-Modifier" -i dev -a bumpPatch
22.03.28.1-dev

```

# Expected behaviour
Return version with choosen modifier value

```
$ docker run --rm relizaio/versioning -s "0Y.0M.0D.Micro-Modifier" -i dev -a bumpPatch -v "22.03.28.2-dev"
22.03.28.3-dev
```

**Comment by taleodor** [comment]
Fixed in 2021.12.3

**Comment by frzb** [comment]
Thank you so very much.

🇺🇦 Нет войне! 🇺🇦
